src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacyContent.tsx

```diff
@@ -20,6 +20,7 @@ type Props = {
 
 class ProjectDataPrivacyContent extends AsyncView<Props> {
   static contextTypes = {
+    organization: PropTypes.object,
```

Comment:
```suggestion
    organization: SentryTypes.Organization,
```

Don't forget import if not already there :)